File: code-examples/python/products/create_variants.py
import http.client

conn = http.client.HTTPConnection("api.marketcloud.it")

payload = "[{ \"price\":13.5, \"name\":\"Scala tshirt\", \"description\":\"Tshirt\", \"stock_type\":\"infinite\", \"color\":\"black\", \"size\":\"xl\", \"id\":1}]"

headers = {
    'authorization': "YourPublicKey:YourToken",
    'accept': "application/json",
    'content-type': "application/json"
    }

conn.request("POST", "/v0/products/:id/variants", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: code-examples/csharp/products/create_variants.cs
var client = new RestClient("http://api.marketcloud.it/v0/products/:id/variants");
var request = new RestRequest(Method.POST);
request.AddHeader("content-type", "application/json");
request.AddHeader("accept", "application/json");
request.AddHeader("authorization", "YourPublicKey:YourToken");
request.AddParameter("application/json", "[{ \"price\":13.5, \"name\":\"Scala tshirt\", \"description\":\"Tshirt\", \"stock_type\":\"infinite\", \"color\":\"black\", \"size\":\"xl\", \"id\":1}]", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);